packages/pages/src/upgrade/upgrade.ts

```diff
@@ -12,8 +25,20 @@ interface UpgradeArgs {
 
 const handler = async (args: UpgradeArgs) => {
   const scoped = { scope: args.scope || "" };
+  const source = path.resolve(scoped.scope);
   const projectStructure = await ProjectStructure.init(scoped);
-  await updatePages(projectStructure);
+  await updateSitesComponents(source);
+  await updateDevDependencies(source);
+  checkLegacyMarkdown(source);
+  replacePagesSlashComponentsImport(source);
```

Comment:
This should be combined with updateSitesComponents

Reply:
In the future, I think we will likely want to be updating the import but not updating sites-components in package.json, because sites-components will no longer exist as it is replaced with pages-components

Reply:
Right. This will need to be part of the (either): `updateToUseSitesComponents` (current) or `updateToUsePagesComponents` (latest one)